src/server/middleware/security/utils.go

```diff
@@ -16,3 +19,13 @@ func bearerToken(req *http.Request) string {
 	}
 	return strings.TrimSpace(token[1])
 }
+
+// FromJobservice detects whether this request is from jobservice.
+func FromJobservice(req *http.Request) bool {
```

Comment:
it needs to narrow down to tag retention job, otherwise, other jobs which call the REST API to delete artifact will be impacted.

Reply:
updated.